adam/adam_6224.py

```diff
@@ -0,0 +1,628 @@
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    AnalogOutput_1 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    AnalogOutput_2 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    AnalogOutput_3 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    SafetyValue_0 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    SafetyValue_1 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    SafetyValue_2 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    SafetyValue_3 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+    
+    StartupValue_0 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    StartupValue_1 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    StartupValue_2 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    StartupValue_3 = attribute(
+        dtype='double',
+        access=AttrWriteType.READ_WRITE,
+        format='%2.4f'
+    )
+
+    TypeCode_0 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ_WRITE,
+    )
+
+    TypeCode_1 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ_WRITE,
+    )
+
+    TypeCode_2 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ_WRITE,
+    )
+
+    TypeCode_3 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ_WRITE,
+    )
+
+    Status_0 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    Status_1 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    Status_2 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    Status_3 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    EventStatus_0 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    EventStatus_1 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    EventStatus_2 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    EventStatus_3 = attribute(
+        dtype='str',
+        access=AttrWriteType.READ,
+    )
+
+    # ---------------
+    # General methods
+    # ---------------
+
+    def init_device(self):
+        Device.init_device(self)
+        # PROTECTED REGION ID(ADAM6224.init_device) ENABLED START #
+        self.set_state(DevState.ON)
```

Comment:
I wouldn't use `ON` state as the default if you don't connect to an ADAM at initialisation. Please change that to `STANDBY` and update the status accordingly as well.